### DISCUSSION TITLE
Mame bug?

### DISCUSSION THREAD

**Discussion by LastNinja2** [OnionUI/Onion#527] [Q&A] [answered]
Hi everyone,
usually pressing Menu I can exit a game, but with a lot of Mame games it doesn't happen, so I have to enter retroarch -> close content -> quit retroarch.
Is this a bug or there's another way?

**Comment by Olywa** [comment, marked as answer]
The Mame core is a little buggy with states and resuming, I think because of the differences between games and the sheer number of machines it's actually emulating. It's a known issue I think, I and others have seen this too.

**Reply by LastNinja2** [reply]
Got it, good to know, thanks
